blazor/datagrid/events.md

```diff
@@ -2513,7 +2511,53 @@ The events should be provided to the datagrid using **GridEvents** component. Wh
         public double? Freight { get; set; }
     }
 
-    public void RowDropHandler(RowDragEventArgs<Order> args)
+    public void RowDropHandler(RowDroppedEventArgs<Order> args)
+    {
+        // Here, you can customize your code.
+    }
+}
+```
+
+## RowDropping
+
+`RowDropping` event triggers when the row elements are dropping on the target element. You can cancel the dropping action using this event.
```

Comment:
Insert a separate note like ' to prevent the dropping action kindly use args.Cancel argument of RowDropping event.